pyproject.toml

```diff
@@ -1,3 +1,113 @@
+# Package ######################################################################
+
+[build-system]
+requires = ["setuptools >= 61.0.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "gymnasium"
+description = "A standard API for reinforcement learning and a diverse set of reference environments (formerly Gym)."
+readme = "README.md"
+requires-python = ">= 3.7"
+authors = [{ name = "Farama Foundation", email = "contact@farama.org" }]
+license = { text = "MIT License" }
+keywords = ["Reinforcement Learning", "game", "RL", "AI", "gymnasium"]
+classifiers = [
+    "Development Status :: 4 - Beta",  # change to `5 - Production/Stable` when ready
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    'Intended Audience :: Science/Research',
+    'Topic :: Scientific/Engineering :: Artificial Intelligence',
+]
+dependencies = [
+    "numpy >=1.21.0",
+    "jax-jumpy >=0.2.0",
+    "cloudpickle >=1.2.0",
+    "importlib-metadata >=4.8.0; python_version < '3.10'",
+    "typing-extensions >=4.3.0; python_version == '3.7'",
+    "gymnasium-notices >=0.0.1",
+    "shimmy >=0.1.0,<1.0",
+]
+dynamic = ["version"]
+
+[project.optional-dependencies]
+# Update dependencies in `all` if any are added or removed
+atari = ["shimmy[atari] >=0.1.0,<1.0"]
+accept-rom-license = ["autorom[accept-rom-license] ~=0.4.2"]
+box2d = ["box2d-py ==2.3.5", "pygame ==2.1.0", "swig ==4.*"]
```

Comment:
We have updated pygame to `2.1.2.dev8` I believe which adds support for python 3.11